Fix summary of AGPL in README.md (#3067)
quickwit-oss/quickwit: README.md

### What license does Quickwit use?

> Quickwit is open-source under the GNU Affero General Public License Version 3 - AGPLv3. Fundamentally, this means that you are free to use Quickwit for your project, as long as you don't modify Quickwit. If you do and you are distributing your modified version to the public, you have to make the modifications public.

Reviewer: ```suggestion
Quickwit is open-source under the GNU Affero General Public License Version 3 - AGPLv3. Fundamentally, this means you are free to use Quickwit for your project if you don't modify Quickwit. However, if you do and you are distributing your modified version to the public, you have to make the modifications public.
```
